concrete/attributes/select/controller.php

```diff
@@ -525,13 +529,22 @@ public function action_load_autocomplete_values()
         $values = [];
         // now, if the current instance of the attribute key allows us to do autocomplete, we return all the values
         if ($this->akSelectAllowOtherValues) {
-            $options = $this->getOptions($_GET['q']);
+            $term = $this->request->request->get("term");
+            $options = $this->getOptions($term);
             foreach ($options as $opt) {
                 $o = new \stdClass();
-                $o->id = 'SelectAttributeOption:' . $opt->getSelectAttributeOptionID();
+                $o->value = 'SelectAttributeOption:' . $opt->getSelectAttributeOptionID();
                 $o->text = $opt->getSelectAttributeOptionValue(false);
                 $values[] = $o;
             }
+
+            // this is required so bootstrap-select can add the new value to its list
+            // if (count($values) === 0) {
```

Comment:
`if ($values === [])` is much faster :wink:

Reply:
@mlocati Good suggestion. I never thought of that

Reply:
oh wait, this is a comment, something I was supposed to remove altogether....